static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -1488,6 +1424,85 @@ class IssueRuleEditor extends AsyncView<Props, State> {
 
 export default withOrganization(withProjects(IssueRuleEditor));
 
+export const findIncompatibleRules = (setIncompatible: SetIncompatibleFunction, rule) => {
```

Comment:
Let me know if there's a util file I can put this in or if it should be in its own file